lib/matplotlib/axes/_base.py

```diff
@@ -1705,14 +1705,23 @@ def axis(self, *v, **kwargs):
             ymin, ymax = self.set_ylim(ymin, ymax, emit=emit, auto=auto)
             return xmin, xmax, ymin, ymax
 
-        v = v[0]
-        if len(v) != 4:
-            raise ValueError('v must contain [xmin xmax ymin ymax]')
+        args = args[0]
```

Comment:
As written, I would leave `v` from here on. `v = args[0]`. We are switching from the argument list `args` to a single argument. It's bad style to use the same name for both.